kyuubi-main/src/test/scala/org/apache/kyuubi/server/MiniYarnService.scala

```diff
@@ -104,6 +109,24 @@ class MiniYarnService(name: String) extends AbstractService(name) {
     }.toMap
   }
 
+  private def saveHadoopConf(): Unit = {
```

Comment:
Can org.apache.hadoop.conf.Configuration#writeXml(java.io.OutputStream) be used here？

Reply:
thanks